### DISCUSSION TITLE
New Request: Mobile App, Desktop App, IDE App

### DISCUSSION THREAD

**Discussion by NewMan2024** [danny-avila/LibreChat#3948] [Feature Requests & Suggestions]
Hello
Initializing a request for a cross-platform app or native app, for mobile and desktop. It is better to have flexible architecture to have alternatives such as mobile and desktop. what do you think?

**Comment by sushink70** [comment]
I am thinking about this. For now fork the project and run the LibreChat app inside a tauri desktop application.
